tests/sentry/rules/conditions/test_tagged_event.py

```diff
@@ -88,3 +88,25 @@ def test_does_not_contain(self):
             data={"match": MatchType.NOT_CONTAINS, "key": "logger", "value": "bar.foo"}
         )
         self.assertPasses(rule, event)
+
+    def test_is_set(self):
+        event = self.get_event()
+        rule = self.get_rule(data={"match": MatchType.IS_SET, "key": "logger", "value": "whatever"})
+        self.assertPasses(rule, event)
+
+        rule = self.get_rule(
+            data={"match": MatchType.IS_SET, "key": "missing", "value": "whatever"}
+        )
+        self.assertDoesNotPass(rule, event)
+
+    def test_is_not_set(self):
+        event = self.get_event()
+        rule = self.get_rule(
+            data={"match": MatchType.NOT_SET, "key": "logger", "value": "whatever"}
```

Comment:
Why do you need to provide a value if it is not used for the match rule?

Reply:
Mainly to avoid returning `False` if not all three values are passed, to respect the [original design](https://github.com/getsentry/sentry/blob/master/src/sentry/rules/conditions/tagged_event.py#L52). But I agree that it shouldn't be required.

Reply:
As far as I understand, it should be safe to remove that requirement for `value` to be truthy. As any further comparison would just be evaluated to false if `value` is an empty string.

I updated my commit to remove that restriction, and updated the tests accordingly.